apps/system/style/window.css

```diff
@@ -389,7 +389,7 @@ html[dir="rtl"] .appWindow.invoked {
   opacity: 1;
 }
 
-.appWindow.in-task-manager.overlay.no-screenshot > .identification-overlay {
+#screen.cards-view .appWindow.in-task-manager.overlay.no-screenshot > .identification-overlay {
```

Comment:
when going to the cardview from an app freshly launched the white overlay was flashing (because the app doesn't have a screenshot yet)